src/charm.py

```diff
@@ -115,8 +128,40 @@ def __init__(self, *args) -> None:
         )
         self.info_provider = OathkeeperInfoProvider(self)
 
+        self.metrics_endpoint = MetricsEndpointProvider(
+            self,
+            relation_name=PROMETHEUS_SCRAPE_RELATION_NAME,
+            jobs=[
+                {
+                    "metrics_path": PROMETHEUS_METRICS_PATH,
+                    "static_configs": [
+                        {
+                            "targets": [f"*:{OATHKEEPER_METRICS_PORT}"],
+                        }
+                    ],
+                }
+            ],
+        )
+
+        self.loki_consumer = LogProxyConsumer(
+            self,
+            log_files=[str(LOG_PATH)],
+            relation_name=LOKI_PUSH_API_RELATION_NAME,
+            container_name=self._container_name,
+        )
```

Comment:
We should use LogForwarder (see https://github.com/canonical/openfga-operator/blob/main/src/charm.py#L111)